docs/purchasing/parts.json

```diff
@@ -101,6 +101,57 @@
       }
     ]
   },
+  {
+    "pn": "ec-mega2x1",
+    "notes": "Mega-Fit 2x1",
+    "manufacturer": "Molex",
+    "mpn": "1716920102",
+    "avg_price": 0.47,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "DigiKey",
+        "spn": "WM10385-ND",
+        "price": 0.47,
+        "price_units": 1,
+        "url": "https://www.digikey.com/en/products/detail/molex/1716920102/4515271"
+      }
+    ]
+  },
+  {
+    "pn": "ec-micro1x4",
+    "notes": "Micro-Fit 1x4",
+    "manufacturer": "Molex",
+    "mpn": "0436450408",
+    "avg_price": 0.62,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "DigiKey",
+        "spn": "WM13985-ND",
+        "price": 0.62,
+        "price_units": 1,
+        "url": "https://www.digikey.com/en/products/detail/molex/0436450408/4481616"
+      }
+    ]
+  },
+  {
+    "pn": "ee-blower-tkfan",
+    "notes": "TKFAN Blower 12V",
+    "manufacturer": "TKFAN",
+    "mpn": "BA7050H12B",
+    "avg_price": 35.00,
+    "price_unit": "",
+    "sources": [
+      {
+        "supplier": "TKFAN",
+        "spn": "BA7050H12B",
+        "price": 35.00,
+        "price_units": 1,
+        "url": "http://tkfan.com/Hot_Sell/ba7050-22.html"
```

Comment:
Link here appears to be to the 24V version, which is not the one we are using?
Even if there is a page for the right version on the manufacturer's site, we should probably include a link to the one that can be bought on AliExpress, as that is practically where we got ours from, right?